src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+$days-count: 28, 29, 30, 31;
+$cell-size: 100px;
+$days-of-the-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
+  box-sizing: border-box;
+  width: 7 * $cell-size + 6px;
```

Comment:
The calculation for the width is incorrect. The expression `7 * $cell-size + 6px` will not be evaluated as expected because the multiplication needs to be wrapped in parentheses to ensure correct order of operations. It should be `(7 * $cell-size) + 6px`.